jobs/event_logger/monit

```diff
@@ -0,0 +1,18 @@
+<%= 
+
+require 'json'
+
+
+monit = {
+  :"processes" => []
```

Comment:
Why do we symbolize the strings here? Might read cleaner to just change this and similar to be more like:
```ruby
{
  proccesses: []
}
```